ports/itpp/fix_include.patch

```
@@ -0,0 +1,16 @@
+diff --git a/itpp/CMakeLists.txt b/itpp/CMakeLists.txt
+index e8c78d3..2aa549c 100644
+--- a/itpp/CMakeLists.txt
++++ b/itpp/CMakeLists.txt
+@@ -42,8 +42,10 @@ include_directories ( ${CMAKE_BINARY_DIR}
+                       ${CMAKE_SOURCE_DIR}
+                       ${BLAS_INCLUDES}
+                       ${LAPACK_INCLUDES}
+-                      ${FFT_INCLUDES}
+                     )
++if(FFT_FOUND)
++    include_directories(${FFT_INCLUDES})
++endif()
```

```suggestion
+if(FFT_FOUND)
+    include_directories(${FFT_INCLUDES})
+else
+    message(FATAL_ERROR "failed to find FFT")
+endif()
```

this should fail if FFT is not found, so that the resulting package doesn't depend on whether FFT is found or not.

---

It already check the fft library and give a waring in the main CMakeLists.txt file which under the root directory of itpp, so I don't this is needed.